docs/source/developer/accessibility.rst

```diff
@@ -0,0 +1,172 @@
+in Lumino, it's better to fix it in Lumino because then the fix will be absorbed
+in more places. However, for that same reason, because Lumino is used by more
+codebases than just JupyterLab—specifically, by JupyterLab extensions—one should
+be careful making changes to Lumino that might break downstream
+consumers/extensions. So an additional rule of thumb is: if you can't make the
+fix in Lumino without breaking dependants, then it might be better to make the
+fix in JupyterLab. In this case, you might take a two-track approach, where you
+fix the accessibility issue in JupyterLab and also submit a breaking fix in
+Lumino that targets a future, breaking version of Lumino.
+
+Testing
+-------
+
+If you fix an accessibility issue in the source code but you don't add a test
+with your fix, then there's a strong chance that your fix will be undone by
+accident by some future changes to the codebase.
+
+Sometimes it's straightforward to unit-test an accessibility fix, such as when
+`enabling keyboard shortcuts on a toolbar button
+<https://github.com/jupyterlab/jupyterlab/pull/5769>`__. But often it's
+difficult to unit-test accessibility fixes.
+
+Therefore there is an effort underway to use Playwright to write user level
+`accessibility tests to JupyterLab
+<https://github.com/Quansight-Labs/jupyter-a11y-testing/tree/main/testing/jupyterlab>`__.
+Here's how you can use it in your development process, once you've identified an
+accessibility issue and how to fix it:
+
+1. Fork the _jupyter-a11y-testing_ repo.
+2. Create a new git branch. Add a test using one of the existing JupyterLab
+   regression tests in the repo as a model. The idea is that this test should
+   fail without your accessibility fix but pass after your fix is merged.
+3. On your fork of jupyter-a11y-testing, using the instructions at the link
+   above, manually dispatch the workflow to run your new test and verify that it
+   fails.
+4. Open a PR on Lumino or JupyterLab or both that contains your accessibility
+   fix.
+5. Using your _jupyter-a11y-testing_ fork, manually dispatch the workflow again,
+   but this time configure the workflow so that it builds JupyterLab with your
+   code fixes applied. For example, if your GitHub username is `a11ydev` and the
+   name of your JupyterLab branch is `jl-a11y-fix`, then in the workflow form
+   you would put `a11ydev/jupyterlab` as the repo and `a11y-fix` as the
+   ref/branch/tag/SHA. If instead (or also) you have a fix for Lumino in a
+   branch named `lm-a11y-fix`, then you would put `a11ydev/lumino` in the field
+   for the external package repo and the name of your Lumino branch in the field
+   for the external package ref.
+
+Altogether, the form will look something like this before you submit it:
+
+- JupyterLab repo: `a11ydev/jupyterlab`
+- JupyterLab ref: `jl-a11y-fix`
+- External package reo: `a11ydev/lumino`
+- Extrenal package ref: `lm-a11y-fix`
+
```

Comment:
This may be out of scope. Feel free to ignore if desired.

```suggestion

Manual Testing
---------------

Many accessibility issues become most clear when caught in action. Automated testing can be good at this, but you may also be able to find the impacts of accessibility of a contribution. 

One of the most approachable ways to try this is to try and complete a task related to your fix or contribution using an accessibility accommodation or setting. Common options include:

- Using a `screen reader`<https://en.wikipedia.org/wiki/Screen_reader>__. 
- Zooming the page up to 400% via your browser.
- Unplugging or not using your mouse. Navigate only with the keyboard.

While testing, take note of what happens and compare it to what you can do to complete the task without your chosen accessibility accommodation. If there is anything you cannot complete, then you have a blocking accessibility issue. Even though your use of assistive tech or an accessibility accommodation will likely differ from someone who uses them regularly, knowing the results is helpful to tell if JupyterLab is behaving as you expect.
```

Reply:
Thank you, this is great! I added this text but I kept the "manual testing" heading under the overall "testing" heading